chemprop/models/model.py

```diff
@@ -85,6 +88,9 @@ def __init__(
         self.bn = nn.BatchNorm1d(self.message_passing.output_dim) if batch_norm else nn.Identity()
         self.predictor = predictor
 
+        self.graph_transform = graph_transform
```

Comment:
is it possible to place this inside the `MessagePassing` class? That seems like a more natural place for it

Reply:
also:
```suggestion
        self.graph_transform = graph_transform if graph_transform is not None else nn.Identity()
```